source/renderer/app/stores/ada/NodeUpdateStore.js

```diff
@@ -30,9 +30,9 @@ export default class NodeUpdateStore extends Store {
     setInterval(this.refreshNextUpdate, NODE_UPDATE_POLL_INTERVAL);
   }
 
-  @action refreshNextUpdate = () => {
+  @action refreshNextUpdate = async () => {
     if (this.stores.networkStatus.isSynced) {
-      this.nextUpdateRequest.execute();
+      await this.nextUpdateRequest.execute();
       if (this.nextUpdateRequest.result && !this.isUpdateAvailable &&
```

Comment:
@DominikGuzei shouldn't we now wrap the code after await in the `runInAction` block?

Reply:
Seems to be supported by mobx now: https://mobx.js.org/best/actions.html#async-await

Reply:
Perpahs I was misunderstood. Don't you think the lines 37-39 should be turned into this:
```
runInAction(() => {
        this.isUpdateAvailable = true;
        this.isNotificationExpanded = true;
        this.updateVersion = this.nextUpdateRequest.result.version;
})
```
...this is the recommended way from the docs: https://mobx.js.org/best/actions.html#async-await
Or am I missing something? :)

Reply:
Ah true … i missed the last sentence! 😉